files/root/grafana_tools/bulk_alert/alert_toggle.sh

```diff
@@ -4,6 +4,7 @@ action=$1
 grouping=$2
 tag_or_id=$3
 
+pushd $(dirname "$0")
```

Comment:
Can you explain why this was added?

I'm sure it is probably all good, but just curious why the change.

Reply:
Same reason for https://github.com/jdmaloney/grafana_tools/issues/2, in that the script was breaking since it was running in a different directory when ran via cron.

The alert_toggle.sh script had more than just a single `source ./config`, since it was also loading the json files from `.`, so I opted to have the script pushd and popd back instead of editing the path for each of those lines.

I did see that version of grafana_tools, and we also have another copy over in https://git.ncsa.illinois.edu/ici-monitoring/grafana-tools. So wasn't sure which is our authoritative copy. They have separate commit histories so syncing them might be a bit more ugly than usual. Since one is in JD's personal github space maybe best to use our gitlab version as authoritative.

Reply:
OK, that makes sense.  If we're doing that, then let's also:
- update `~/grafana_tools/backup_and_sync/sync_grafana.sh` to no longer require the `cd` in its `alert_toggle.sh` lines
- apply the same fix to `grafana_tools/ldap_sync/sync.sh` and its CRONs from puppet
- apply the same fix to `grafana_tools/backup_and_sync/sync_grafana` and its CRONs from puppet

I thought JD told me that https://github.com/jdmaloney/grafana_tools was what should be used, but now I'm questioning which is most recent. 😞